services/storage/src/simcore_service_storage/handlers_files.py

```diff
@@ -314,8 +342,10 @@ async def is_completed_upload_file(request: web.Request):
     )
 
 
-@routes.delete(f"/{api_vtag}/locations/{{location_id}}/files/{{file_id}}", name="delete_file")  # type: ignore
-async def delete_file(request: web.Request):
+@routes.delete(
+    f"/{api_vtag}/locations/{{location_id}}/files/{{file_id}}", name="delete_file"
+)
+async def delete_file(request: web.Request) -> NoReturn:
```

Comment:
``NoReturn`` ... interesting! 🤔 
Nice, so it differentiates between returning a None or not returning anything at all (i.e. always raises) 

https://peps.python.org/pep-0484/#the-noreturn-type